src/conversation.js

```diff
@@ -107,11 +107,12 @@ module.exports = AV.Object.extend('_Conversation', {
    *     It could be a raw string, or an object with a `toJSON` method, like a
    *     realtime SDK's Message object. See more: {@link https://leancloud.cn/docs/realtime_guide-js.html#消息}
    * @param {Boolean} [options.transient] Whether send this message as transient message or not.
+   * @param {String[]} [options.toClientIds] Clients' ids to send to. This option can be used only in system conversation.
```

Comment:
Clients' ids to send to -> ids of clients to send to

in system conversation -> with ?